modules/yieldlabBidAdapter.js

```diff
@@ -39,6 +41,15 @@ export const spec = {
 
     _each(validBidRequests, function (bid) {
       adslotIds.push(bid.params.adslotId)
+      let sizes = new Set()
```

Comment:
I think mutating function arguments is not a good idea when the same logic could be expressed as a pure function. I would prefer something like this:

```
const sizes = extractSizes(bid)

if (sizes.size > 0) {
    adslotSizes.push(bid.params.adslotId + ':' + [...sizes].join('|'))
}
```